src/v2/Apps/Partner/Components/PartnerHeader/index.tsx

```diff
@@ -32,58 +33,62 @@ export const PartnerHeader: React.FC<PartnerHeaderProps> = ({ partner }) => {
     partner && partner.type !== "Auction House" && !!partner.profile
 
   return (
-    <GridColumns gridRowGap={2} my={[2, 4]}>
-      <Column span={[12, 10]}>
-        <Flex>
-          {partner.profile?.icon && (
-            <Flex mr={2}>
-              <RouterLink
-                to={partner.href}
-                style={{ display: "flex", textDecoration: "none" }}
-              >
-                <HeaderImage
-                  src={partner.profile.icon.cropped.src}
-                  srcSet={partner.profile.icon.cropped.srcSet}
-                  alt={partner.name}
-                  width={[60, 80]}
-                  height={[60, 80]}
-                />
-              </RouterLink>
-            </Flex>
-          )}
-          <Box>
-            <Text variant="largeTitle">
-              <RouterLink
-                style={{
-                  textDecoration: "none",
-                }}
-                to={partner.href}
-              >
-                {partner.name}
-              </RouterLink>
-            </Text>
-            {hasLocations && (
-              <Text color={color("black60")} variant="text">
-                <PartnerHeaderAddress {...partner.locations} />
-              </Text>
+    <>
+      <PartnerHeaderImage profile={partner.profile} />
+
+      <GridColumns gridRowGap={2} my={[2, 4]}>
+        <Column span={[12, 10]}>
+          <Flex>
+            {partner.profile?.icon && (
+              <Flex mr={2}>
+                <RouterLink
+                  to={partner.href}
+                  style={{ display: "flex", textDecoration: "none" }}
+                >
+                  <HeaderImage
+                    src={partner.profile.icon.cropped.src}
+                    srcSet={partner.profile.icon.cropped.srcSet}
+                    alt={partner.name}
+                    width={[60, 80]}
+                    height={[60, 80]}
+                  />
+                </RouterLink>
+              </Flex>
             )}
-          </Box>
-        </Flex>
-      </Column>
-      <Column span={[12, 2]}>
-        {canFollow && (
-          <FollowProfileButton
-            profile={partner.profile}
-            user={user}
-            contextModule={ContextModule.partnerHeader}
-            buttonProps={{
-              variant: "secondaryOutline",
-              width: "100%",
-            }}
-          />
-        )}
-      </Column>
-    </GridColumns>
+            <Box>
+              <Text variant="largeTitle">
+                <RouterLink
+                  style={{
+                    textDecoration: "none",
+                  }}
+                  to={partner.href}
+                >
+                  {partner.name}
+                </RouterLink>
+              </Text>
+              {hasLocations && (
+                <Text color={color("black60")} variant="text">
```

Comment:
You don't need to use this color function. just `"black60" works fine here. In general avoid the `color` function everywhere.